cmd/git-chglog/config.go

```diff
@@ -243,6 +262,13 @@ func (config *Config) normalizeStyleOfBitbucket() {
 	config.Options = opts
 }
 
+func or_value(str1 string, str2 string) string {
```

Comment:
Lint, prefer camel over snake case.

```suggestion
func orValue(str1 string, str2 string) string {
```